spec.md

```diff
@@ -356,7 +356,7 @@ Service Brokers MAY add, remove or modify (metadata, plans, etc.) the list of
 services from previous queries.
 
 When determining what, if anything, has changed on a Service Broker, the
-Platform MUST use the `id` of the resources (services or plans) as the only
+Platform MUST use the `id` of the resources (Service Offerings or Plans) as the only
```

Comment:
We define **Service Plan** rather than just **Plan** in the [Notations and Terminology](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md#notations-and-terminology) section

Reply:
I'm ok with changing Plan to Service Plan, but right now we're not consistent on its use.